src/game/client/draw/drawable.cpp

```diff
@@ -2098,3 +2101,295 @@ void Drawable::Set_Stealth_Look(StealthLookType look)
+                        m_drawableLocoInfo->m_accelerationRollRate = m_drawableLocoInfo->m_accelerationRollRate
+                            - lateral_accel_coef * (-direction->y * prev_accel.x + direction->x * prev_accel.y);
+                    }
+
+                    if (m_drawableLocoInfo->m_accelerationPitch > deacel_pitch_limit) {
+                        m_drawableLocoInfo->m_accelerationPitch = deacel_pitch_limit;
+                    } else if (-accel_pitch_limit > m_drawableLocoInfo->m_accelerationPitch) {
+                        m_drawableLocoInfo->m_accelerationPitch = -accel_pitch_limit;
+                    }
+
+                    if (m_drawableLocoInfo->m_accelerationRoll > deacel_pitch_limit) {
+                        m_drawableLocoInfo->m_accelerationRoll = deacel_pitch_limit;
+                    } else if (-accel_pitch_limit > m_drawableLocoInfo->m_accelerationRoll) {
+                        m_drawableLocoInfo->m_accelerationRoll = -accel_pitch_limit;
+                    }
+
+                    xform.m_totalZ = 0.0f;
+                    float major_radius = object->Get_Geometry_Info().Get_Major_Radius();
+                    float minor_radius = object->Get_Geometry_Info().Get_Minor_Radius();
+                    float offset1 = GameMath::Sin(xform.m_totalPitch - pitch) * major_radius;
+                    float offset2 = GameMath::Sin(xform.m_pitchRate - roll) * minor_radius;
+
+                    if (has_suspension) {
+                        m_drawableLocoInfo->m_wheelInfo.m_framesAirborne =
+                            m_drawableLocoInfo->m_wheelInfo.m_framesAirborneCounter;
+                        m_drawableLocoInfo->m_wheelInfo.m_framesAirborneCounter = 0;
+                        TWheelInfo wheel_info = m_drawableLocoInfo->m_wheelInfo;
+                        PhysicsTurningType turning = physics->Get_Turning();
+
+                        if (turning == TURN_NEGATIVE) {
+                            wheel_info.m_wheelAngle = -wheel_turn_angle;
+                        } else if (turning == TURN_POSITIVE) {
+                            wheel_info.m_wheelAngle = wheel_turn_angle;
+                        } else {
+                            wheel_info.m_wheelAngle = 0.0f;
+                        }
+
+                        if (physics->Get_Forward_Speed_2D() < 0.0f) {
+                            wheel_info.m_wheelAngle = -wheel_info.m_wheelAngle;
+                        }
+
+                        m_drawableLocoInfo->m_wheelInfo.m_wheelAngle =
+                            (wheel_info.m_wheelAngle - m_drawableLocoInfo->m_wheelInfo.m_wheelAngle) / 10.0f
+                            + m_drawableLocoInfo->m_wheelInfo.m_wheelAngle;
+                        float offset_mult = 0.9f;
+
+                        if (offset1 < 0.0f) {
+                            wheel_info.m_frontLeftHeightOffset = (offset1 / 3.0f + offset1 / 2.0f) * offset_mult;
```

Comment:
ifyou are just gonna reuse left then reuse for all where applicable

Reply:
fixed.